### PR TITLE
(PC-6138) Implement the creditCeiling component

### PR DESCRIPTION
**Pull request by blixit**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-6138

## Checklist

I have:

- [x] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [x] Made sure my feature is working on the relevant real / virtual devices.
- [x] Written **unit tests** for my feature.
- [x] Added new reusable components to the AppComponents page and communicate to the team on slack
- [x] Added a **screenshot** for UI tickets.
- [x] Attached a **ticket number** for any added TODO/FIXME \
       (for tech tasks, give the best context about the TODO resolution: what/who/when).

## Screenshots

| \*iOS - [Phone name] | \*Android - [Phone name] | Tablet - [Phone name] | Little - [Phone name] |
| -------------------- | :----------------------: | --------------------: | --------------------: |
| image | image |  image  |                       |

## Deploy hard

If native code (ios/android) was modified, **after** the PR is merged, on the master branch, upgrade the **app version** (+1 patch):

- if you want an hard deployment of the testing environment, use `yarn trigger:testing:deploy`

### CODE REVIEW

File: src/ui/components/bars/CreditCeiling.test.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import { CreditCeiling } from './CreditCeiling'
+
+describe('CreditCeiling', () => {
+  it('should render anything when the max is negative', () => {
+    const { toJSON } = render(
+      <CreditCeiling amount={155} max={-1} type={'physical'} depositVersion={1} />
+    )
+    expect(toJSON()).toBeNull()
+  })
+  it('should render anything when the max is negative', () => {
```
Comment (AnoukHello) [inline src/ui/components/bars/CreditCeiling.test.tsx:13]:
```suggestion
  it('should render correctly when the max is positive', () => {
```

File: src/ui/components/bars/CreditCeiling.test.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { render } from '@testing-library/react-native'
+import React from 'react'
+
+import { CreditCeiling } from './CreditCeiling'
+
+describe('CreditCeiling', () => {
+  it('should render anything when the max is negative', () => {
```
Comment (AnoukHello) [inline src/ui/components/bars/CreditCeiling.test.tsx:7]:
```suggestion
  it('should not render when the max is negative', () => {
```

File: src/ui/components/bars/CreditCeiling.test.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    const { toJSON } = render(
+      <CreditCeiling amount={155} max={-1} type={'physical'} depositVersion={1} />
```
Comment (AnoukHello) [inline src/ui/components/bars/CreditCeiling.test.tsx:9]: Si le montant est négatif il se passe quoi ?

Comment (blixit) [reply src/ui/components/bars/CreditCeiling.test.tsx:9]: La barre n'est pas affichée.

- si max est 0, le calcul est impossible.
- si max est négatif, le calcul est incohérent.

**Review by AnoukHello — APPROVED** [review]

**Pull request merged**